src/golang/lib/models/shared/service.go

```diff
@@ -85,6 +85,31 @@ func GetRelationalDatabaseIntegrations() map[Service]bool {
 		Athena:       true,
 		MongoDB:      true,
 	}
+
+	_, ok := relationalIntegrations[service]
+	return ok
+}
+
+func IsDatabaseIntegration(service Service) bool {
+	if IsRelationalDatabaseIntegration(service) {
+		return true
+	}
+
+	return service == MongoDB
+}
+
+func IsComputeIntegration(service Service) bool {
+	computeIntegrations := map[Service]bool{
+		Airflow:    true,
+		Lambda:     true,
+		Conda:      true,
+		Databricks: true,
+		Kubernetes: true,
+		Spark:      true,
+	}
```

Comment:
I think we can make this a variable outside of this function at the beginning of `service.go`?